genie.js

```diff
@@ -37,8 +37,25 @@
     }
 
     Wish.prototype.isEqualTo = function(objectToMatch) {
-        assert(this.name, this.subject === objectToMatch)
+        assert(this.name, this.subject.toString() === objectToMatch.toString())
     }
 
+    freezeTime = function(time){
+      this.originalDateToStringMethod = Date.toString();
```

Comment:
do you need parens? should it save the orig function rather than its return value? does it matter?